_posts/2021-09-16-data-prepper-roadmap.markdown

```diff
@@ -8,45 +8,46 @@ date:   2021-09-16 01:01:01 -0700
+Data Prepper is a data ingestion component of the OpenSearch Project that pre-processes documents before storing and indexing in OpenSearch. To pre-process documents, Data Prepper allows you to configure a **pipeline** that specifies a ***source***, ***buffers***, a series of ***processors***, and ***sinks*** *(Figure 1)*. Once you have configured a data pipeline, Data Prepper takes care of managing source, sink, and buffer properties and maintaining state across all instances of Data Prepper on which the pipelines are configured. A single instance of Data Prepper can have one or more configured pipelines. A pipeline definition requires at least a ***source*** and ***sink*** attribute to be configured and uses the default buffer and no processor if they are not configured.
 
 ![Data Prepper Basics](/assets/media/blog-images/2021-09-16-data-prepper-roadmap/figure1.png){: .img-fluid}
 
-*Figure 1: Data Prepper Pipelines and attributes*
+*Figure 1: Data Prepper pipelines and attributes*
 
 
-* **Source** is the input component of a pipeline. It defines the mechanism through which a pipeline will consume records. Source component could consume records either by receiving over http/s or reading from external endpoints like Fluent Bit, Beats, OpenTelemetry collectors, Kafka, SQS, CloudWatch etc. 
-* **Buffer** is a temporary store of data and could either be in-memory or disk based. The default buffer will be an in-memory queue and is the only option available in the initial release.
-* **Sink** is the output component of pipeline. It defines one or more destinations to which a pipeline will publish the records. A sink destination could be either services like OpenSearch, S3, or another pipeline. By using another pipeline as sink, customers can chain multiple Data Prepper pipelines. 
-* **Processors** are intermediary processing units which can filter, transform and enrich the records into the desired format before publishing to the sink. The processor is an optional component of the pipeline, and if not defined the records will be published in the format as defined in the source. You can have more than one processor and they are executed in the order they are defined in the pipeline specification. For example, a pipeline might have one processor that removes a field from the document, followed by another processor that renames a field.
+* **Source** is the pipeline's input component. It defines the mechanism through which a pipeline will consume records. The source component could consume records either by receiving over http(s) or reading from external endpoints like Fluent Bit, Beats, OpenTelemetry collectors, Kafka, Amazon SQS, and Amazon CloudWatch. 
```

Comment:
Perhaps change "http(s)" to "HTTP(S)" or "HTTP/HTTPS".

Reply:
Thanks @dlvenable. I've noted this spelling convention to apply it across our documentation.